api/v1/accounts.js

```diff
@@ -102,7 +117,7 @@ router.post('/register', (req, res) => {
           body: {},
           json: true,
         },
-        async (error, resp) => {
```

Comment:
Why was this removed? The `resp` argument should not be removed from the parameters list, since it's used within the function body.

Reply:
Removed because I brought the request and response to the router post section of the script to allow for inquiry of the code prior to the inquiry for username taken.
line 54:
router.post('/register', async (req, res) => {